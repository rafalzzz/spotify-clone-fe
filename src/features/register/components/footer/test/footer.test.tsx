import { render } from '@testing-library/react';
import React from 'react';
import '@testing-library/jest-dom/extend-expect';

import { Footer } from '../';

describe('Footer', () => {
  it('render component without error', async () => {
    const screen = render(<Footer />);
    expect(screen).toMatchSnapshot();
  });
});
