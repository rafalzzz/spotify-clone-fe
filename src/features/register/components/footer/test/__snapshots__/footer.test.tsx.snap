// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render component without error 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer>
        <div
          class="custom-hyperlink login-user-redirection"
          data-testid="custom-hyperlink"
        >
          <span
            class="login-user-redirection__text-before"
          >
            Already have an account?
          </span>
          <a
            class="login-user-redirection__hyperlink"
            data-testid="custom-hyperlink__hyperlink"
            href="/login"
          >
            Login
          </a>
          <span
            class="login-user-redirection__text-after"
          >
            .
          </span>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer>
      <div
        class="custom-hyperlink login-user-redirection"
        data-testid="custom-hyperlink"
      >
        <span
          class="login-user-redirection__text-before"
        >
          Already have an account?
        </span>
        <a
          class="login-user-redirection__hyperlink"
          data-testid="custom-hyperlink__hyperlink"
          href="/login"
        >
          Login
        </a>
        <span
          class="login-user-redirection__text-after"
        >
          .
        </span>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
